test-kotest5/src/main/kotlin/io/micronaut/test/extensions/kotest5/MicronautKotest5Context.kt

```diff
@@ -116,4 +116,7 @@ class MicronautKotest5Context(
         )
     }
 
+  fun close() {
+      applicationContext.close()
```

Comment:
This throws NPE now, the context is closed and nullified by `afterSpecClass`

Reply:
Yeah this isn't the fix, but I know what is now.
I'll update it and add a test.